projects/cashmere-examples/src/lib/cashmere-example.module.ts

```diff
@@ -0,0 +1,358 @@
+
```

Comment:
@joeskeen is this supposed to be checked in?

Reply:
No, this should not be checked in.  It should be deleted.

Reply:
Thanks - I was wondering!

Reply:
Should it be added to .gitignore?

Reply:
I removed it

Reply:
You shouldn't need to add it to .gitignore.  The new example generation process doesn't create this file.

Also, don't forget to push!